### PR TITLE
Add InsightCard component

### PR DESCRIPTION
**Pull request by JoseeWouters**
## Summary

This PR can be summarized in the following changelog entry:

* Adds an InsightCard component.

## Relevant technical choices:

* Adds InsightCard component for the Insight modal

## Test instructions

This PR can be tested by following these steps:

* See test instructions for [the corresponding branch on Premium](https://github.com/Yoast/wordpress-seo-premium/pull/3121)

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes
* [ ] This PR changes the UI in the plugin. I have added the 'UI change' label to this PR.

## Quality assurance

* [x] I have tested this code to the best of my abilities
* [ ] I have added unittests to verify the code works as intended

Fixes P1-228

### CODE REVIEW

File: packages/components/src/WordOccurrenceInsights.js [hunk 1]
```diff
@@ -69,12 +79,14 @@ const getExplanation = keywords => {
  * @returns {JSX.Element} Rendered WordList component.
  */
 const WordOccurrenceInsights = ( { words } ) => {
-	const header = <p>{ getExplanation( words ) }</p>;
+	const header = <p className="yoast-field-group__title">{ getHeader() }</p>;
```
Comment (maartenleenders) [inline packages/components/src/WordOccurrenceInsights.js:82]: I think here as well, in stead of adding a title here, we could wrap the WordOccurrences in a `FieldGroup`. In fact, I think we could do some refactoring to wrap the `DataModel` in a `FieldGroup`, although I seem to remember there was some reason for us to keep using the `WordOccurrences`. 

I'd be happy to help with this.

**Review by maartenleenders — CHANGES_REQUESTED** [review]
CR 🚧 

I think there are some suboptimal compositions here, I'd be happy to help fix them.

File: packages/components/src/InsightsCard.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+				<div className="yoast-field-group__title">
+					<b>{ this.props.title }</b>
+					{ this.getHelpIcon() }
+				</div>
+				<div className="yoast-insights-card__content">
+					<p className="yoast-insights-card__score">
+						<span className="yoast-insights-card__amount">{ this.props.amount }</span>
+						{ this.props.unit }
+					</p>
+					<div className="yoast-insights-card__description">{ this.getDescription() }</div>
```
[the remark this answers is not in the document]
Comment (JoseeWouters) [reply packages/components/src/InsightsCard.js]: Clean solution, thanks! Can you explain what exactly this code does?

File: packages/components/src/WordOccurrenceInsights.js (hunk 1 above)
Comment (JoseeWouters) [reply packages/components/src/WordOccurrenceInsights.js:82]: Ok, let's discuss this 👍

**Review by maartenleenders — APPROVED** [review]

**Pull request merged**